pkg/application/defaults.go

```diff
@@ -0,0 +1,47 @@
+package application
+
+import (
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"strings"
+)
+
+var DefaultServiceApp = []Option{
```

Comment:
You could extract duplicate options between the two app types and make them available for custom app types